core/src/main/java/com/linecorp/armeria/server/AnnotatedServiceBindingBuilder.java

```diff
@@ -105,6 +125,16 @@ public AnnotatedServiceBindingBuilder requestConverter(RequestConverterFunction
         return this;
     }
 
+    /**
+     * Adds the given {@link RequestConverterFunction}s to this {@link AnnotatedServiceBindingBuilder}.
+     */
+    public AnnotatedServiceBindingBuilder requestConverters(
+            List<RequestConverterFunction> requestConverterFunctions) {
```

Comment:
- `Iterable<? extends RequestConverterFunction>`
- How about deprecating `requestConverter(RequestConverterFunction)` and adding `requestConverters(RequestConverterFunction...)`?